PubNubSwiftConsole/Classes/UI/CollectionView/Cells/MessageCollectionViewCell.swift

```diff
@@ -37,22 +43,41 @@ struct Message: MessageItem {
 
 class MessageCollectionViewCell: CollectionViewCell {
     private let titleLabel: UILabel
+    private let channelDataLabel: UILabel
+    private let channelLabel: UILabel
     override class var reuseIdentifier: String {
         return String(self.dynamicType)
     }
     override init(frame: CGRect) {
-        titleLabel = UILabel(frame: CGRect(x: 5, y: 0, width: frame.size.width, height: frame.size.height/3))
+        titleLabel = UILabel(frame: CGRect(x: 5, y: 0, width: frame.size.width, height: frame.size.height/4))
+        channelDataLabel = UILabel(frame: CGRect(x: 5, y: 30, width: frame.size.width, height: frame.size.height/4))
+        channelLabel = UILabel(frame: CGRect(x: 5, y: 60, width: frame.size.width, height: frame.size.height/4))
         super.init(frame: frame)
         contentView.addSubview(titleLabel)
-        
+        contentView.addSubview(channelDataLabel)
+        contentView.addSubview(channelLabel)
         contentView.layer.borderWidth = 1
     }
     
     required init?(coder aDecoder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
+    
     func updateStatus(item: MessageItem) {
```

Comment:
it's called `updateStatus` but then you pass in a message, maybe let's call this `updateMessage`?